build.proj

```diff
@@ -53,6 +53,7 @@
       <!-- For the root traversal default filter the OSGroup to the OSEnvironment which is the OS we are running on -->
       <FilterToOSGroup Condition="'$(_OriginalOSGroup)' == ''">$(OSEnvironment)</FilterToOSGroup>
     </Project>
+    <Project Include="layout\dir.proj" />
```

Comment:
Not that it really matters, but should we add a condition here so that we can turn off the layout build if desired? Like we do for tests and packages.